lib/doc/anchor.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2018 Paweł Siemienik
```

Comment:
Personally, I think this shouldn't be in individual files

Reply:
@allube In each file is copyright/author block similar to this, if it's wrong could you write how it should looks like?

Reply:
ah indeed - I had completely missed that. Let's leave that in then.

@guyonroche I've only seen licenses repeated in every file if you want to charge license fees for your IP, but since we use MIT I think there's no need to have this repeated everywhere - it's just noise that needs maintenance. Would you be okay with removing them everywhere?

Reply:
@alubbe  Yeah - We could probably get away with just the LICENCE file with links from the README.md and main excel.js file.
I finally have more spare time now so I can start working through the PRs. I'll test this one tomorrow morning